README.adoc

```diff
@@ -1,5 +1,12 @@
 = devon4j
 
+ATTENTION: You are on the `develop` branch. 
+This has been renamed to `master`. 
+The `develop` branch will not be maintained anymore.
+It is only left here to avoid broken links to existing content.
+Please update links to point to the `master` branch.
+For details look at issue https://github.com/devonfw/devon4j/issues/320[#320].
+
```

Comment:
To whom do you provide this information?
What is the intention why a devonfw customer has to know such internal stuff?

Reply:
This is from isse #320.
It should have prevented this entire PR but still I sometimes had been mislead by old browser history and URL auto-completion leading me to the wrong branch...